src/metrocs/redistricting/StarGeneratorTest.java

```diff
@@ -0,0 +1,45 @@
+package metrocs.redistricting;
+import java.util.ArrayList;
+import static org.junit.Assert.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import org.junit.jupiter.api.Test;
+/**
+ * Tests for objects of type Location in Star Generator.
+ *
+ * @authors  Colin K, Jesus AE, Thomas M, Robert B
+ * @version 20231012.1
+ */
+
+class StarGeneratorTest{
+
+    /** Verifies that locations inside pattern generated are not empty */
```

Comment:
What is the defined behavior for StarGenerator with a parameter of 0?
Is the result empty? null? throws and exception?